2023-11-15 executive spell doc (#2509)
makerdao/community: governance/votes/Executive Vote - November 15, 2023.md

---

> title: Template - [Executive Vote] Spark Proxy Spell, Increase Spark Lend Maximum Debt Ceiling, Launch Project Funding, DAO Resolutions, Whistleblower Bounty, Delegate Compensation - November 15, 2023

Reviewer: I believe it would be more efficient if the title of the compensation included the month, making it easier to search, like "Nov 2023 AD Compensation"

summary: Execute Spark proxy spell, increase the Maximum Debt Ceiling for the Spark Lend D3M, Dai and MKR funding for the Launch Project, DAO resolutions for the HVBank deal, payment of whistleblower bounty, Aligned Delegate compensation for October.
date: 2023-11-15T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] Spark Proxy Spell, Increase Spark Lend Maximum Debt Ceiling, Launch Project Funding, DAO Resolutions, Whistleblower Bounty, Delegate Compensation - November 15, 2023

> The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

Reviewer: ```suggestion
The Governance Facilitators, Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:
- A proxy spell for Spark Lend will be executed to update previously polled parameters.

> - The Spark Lend Dai Direct Deposit Module (D3M) will have its Maximum Debt Ceilling (`line`) increased.

Reviewer: ```suggestion
- The Spark Lend Dai Direct Deposit Module (D3M) will have its Maximum Debt Ceiling (`line`) increased.
```

- Increase the Spark Lend Ethereum rETH and wstETH [supply caps](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps):
	- Increase the rETH supply cap from 60,000 rETH to **80,000 rETH**.
	- Increase the wstETH supply cap from 400,000 wstETH to **800,000 wstETH**.
- Adjust the Spark Lend Ethereum **and** Gnosis Chain ETH [interest rate model](https://docs.aave.com/risk/liquidity-risk/borrow-interest-rate#interest-rate-model) parameters:
	- Base rate: **0%**
	- Optimal Rate: **3.2%**
	- Optimal Utilization: **90%**
	- Maximum Rate: **123.2%**

### Adjust Spark Lend D3M Maximum Debt Ceiling (`line`)

*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmVbrypf#poll-detail)\
*Proposal*: [Forum Post](https://forum.makerdao.com/t/proposal-to-adjust-sparklend-parameters/22542)

If this executive proposal passes, the following change will be made to the Spark Lend Direct Deposit Module Maximum Debt Ceiling:

- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 400 million DAI to **800 million DAI**.

### Launch Project Funds Transfer

> *Authorization*: [Forum Notification](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/6)

Reviewer: I would also include this URL: https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget

Author: It is included on line 83. We don't think duplication is necessary

If this executive proposal passes, the following transfers will be made to the Launch Project as part of the funding defined in the [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-launch-project):

- Transfer **2,200,000 DAI** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
- Transfer **500.00 MKR** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).

### DAO Resolutions Pertaining to HVBank (RWA009-A)

*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmNgKzcG)\
*Proposal*: [Forum Post](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/19)

> If this executive proposal passes, two DAO Resolutions pertaining to the HVBank facility will be approved. These are broken down as follows:

Reviewer: ```suggestion
If this executive proposal passes, a DAO Resolution and Standing Instructions pertaining to the HVBank facility will be approved. These are broken down as follows:
```

Reviewer: I would use the same language Ecosystem used

Author: see: https://github.com/0xldr/community/commit/0cdf553554a882f52c36fc8d4583849797a13265

> If this executive proposal passes, two DAO Resolutions pertaining to the HVBank facility will be approved. These are broken down as follows:

Author: We are unsure. Do you have a suggestion for improved wording? If not, we suggest we proceed with approval.

> If this executive proposal passes, two DAO Resolutions pertaining to the HVBank facility will be approved. These are broken down as follows:

Author: see: https://github.com/0xldr/community/commit/0cdf553554a882f52c36fc8d4583849797a13265

#### Updated Standing Instructions to Escrow Agent

*IPFS link*: https://gateway.pinata.cloud/ipfs/QmWVWXckY482WLTtCFv3x45DFioV1K8mfRM3FVrodqUDud

> ```

Reviewer: I've mentioned this to LB, but it really comes down to personal preference. Including the IPFS text in the Executive Copy might result in losing readers due to its complexity and length. The voters already have access to the link and hash for reading, so it might not be necessary to include the entire content here

Author: We can discuss this later, but we will not be changing this now in the interest of time.

Authorizations:

In accordance with the governance protocols of MakerDAO on the date specified above and pursuant to clause 4.16 of the Articles, this DAO Resolution approves and ratifies the following actions:
5. Subject to the approval of the Verification Agent, the director of the Trust Sponsor is instructed to promptly provide new outgoing instructions to the Escrow Agent, as outlined below (“New Instructions”):

Receiving Bank Routing No: 021000018
ABA Swift IRVTUS3N
Receiving Bank Name: The Bank of New York Mellon
Receiving Bank Address: 240 Greenwich Street, New York NY 10017
Beneficiary Name: Galaxy Digital Trading Cayman LLC
Beneficiary Address: 101 Hudson Street, FL 21 Jersey City, NJ 07302
Beneficiary Account: 8901637157

6. The director of the Trust Sponsor is instructed to take or enter into all required actions and agreements
with third parties solely to facilitate the implementation of the New Instructions.
```

### Whistleblower Bounty

*Authorization*: [Forum Post](https://forum.makerdao.com/t/ads-derecognition-due-to-operational-security-breach/22532)

> If this executive proposal passes, the following whistleblower bounty will be paid to forum user Venice Tree as per [The Atlas](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-operational-security):

Reviewer: I believe this Atlas URL should also be included in the Authorization list

Author: The Atlas is not authorizing the payment. The Facilitator (you) is authorizing the payment.

Author: It is also stated that the action is clearly taking place "per The Atlas" with the relevant link provided

- Transfer **27.78 MKR** from the Pause Proxy to [0xCDDd2A697d472d1e8a0B1B188646c756d097b058](https://etherscan.io/address/0xCDDd2A697d472d1e8a0B1B188646c756d097b058).

### Delegate Compensation for October

*Authorization*: [Atlas 2.6.3](https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-income-and-participation-requirements)\
*Proposal*: [Forum Post](https://forum.makerdao.com/t/october-2023-aligned-delegate-compensation/22732)

> If this executive proposal passes, the following distributions will be made for October Aligned Delegate Compensation, totalling **279.49 MKR**:

Reviewer: ```suggestion
If this executive proposal passes, the following distributions will be made for October Aligned Delegate Compensation, totaling **279.49 MKR**:
```
